consoles/sshXtermIPMI.pm

```diff
@@ -52,6 +54,12 @@ sub reset {
     return;
 }
 
+sub disable {
+    my ($self) = @_;
+    bmwqemu::diag "Try to deactivate IPMI SOL during disable";
+    $self->reset;
+    $self->SUPER::disable;
+}
```

Comment:
add a blank line after that for separation

Reply:
> add a blank line after that for separation

Thank you! Fixed